### PR TITLE
Set up schedule items

### PR DESCRIPTION
**Pull request by andrecardoso**
- Items are parsed from items array attribute or nested rise-playlist-item components

### CODE REVIEW

File: src/rise-playlist.js [hunk 1]
```diff
@@ -1,33 +1,120 @@
...
+
+      if (item["play-until-done"]) {
+        playListItem.setAttribute("play-until-done", item["play-until-done"]);
+      }
+
+      if (item["transition-type"]) {
+        playListItem.setAttribute("transition-type", item["transition-type"]);
+      }
+
+      const element = document.createElement(item.element.tagName);
```
Comment (olegrise) [inline src/rise-playlist.js]: @andrecardoso not sure that it's safe to assume that `item.element` exists. I would check that condition at the very top and skip creation of the "rise-playlist-item" if `item.element` is not there.

Comment (andrecardoso) [reply src/rise-playlist.js]: Good point. I'll filter out invalid items from the array before creating the elements

File: src/rise-playlist.js [hunk 2]
```diff
@@ -1,33 +1,120 @@
...
+          element.style.display = "none";
+        }
+      };
+
+      info.addedNodes.forEach(hideElement);
+
+      const playlistItems = info.addedNodes.filter(node => node.tagName === "RISE-PLAYLIST-ITEM");
+
+      playlistItems.forEach(element => {
+        this.schedule.items.push({duration: parseInt(element.getAttribute("duration"), 10) || 10, playUntilDone: element.hasAttribute("play-until-done"), element});
```
Comment (olegrise) [inline src/rise-playlist.js]: @andrecardoso this is hard to read. Do you mind putting each property of the JSON object into new line?

File: src/rise-playlist.js [hunk 3]
```diff
@@ -56,8 +56,20 @@ export default class RisePlaylist extends RiseElement {
...
+      this.removeChild(this.firstChild);
+    }
+  }
+
   _itemsChanged(items) {
-    items.map(item => {
+    this._removeAllItems();
+
+    const validItems = items.filter(item => {
+      return item.element && item.element.tagName && item.element.tagName !== "";
```
Comment (olegrise) [inline src/rise-playlist.js]: Nice!

**Pull request merged**